bake/bake.go

```diff
@@ -99,7 +99,7 @@ func ListTargets(files []File) ([]string, error) {
 	return dedupSlice(targets), nil
 }
 
-func ReadTargets(ctx context.Context, files []File, targets, overrides []string, defaults map[string]string) (map[string]*Target, map[string]*Group, error) {
+func ReadTargets(_ context.Context, files []File, targets, overrides []string, defaults map[string]string) (map[string]*Target, map[string]*Group, error) {
```

Comment:
Looks like `revive` linter error? Recent revive seems too strict so maybe we can have some `exclude-rules` like this: https://github.com/moby/buildkit/blob/3dedf3c5e72c55e2a61cd98e67e0f461d545b195/.golangci.yml#L61

Reply:
Ah gotcha thanks. I was wondering why we had so many issues. Let me check.